### PR TITLE
Add image upload

### PR DESCRIPTION
**Pull request by YaoLiu93**
For #43 - Image upload

- Adds a lightbox effect upon clicking on the card image. Useful to enlarge any user uploaded image. 
- Adds the ability to upload an image

If no image is uploaded, the source will be our default source.

### CODE REVIEW

File: imports/ui/components/Card.jsx [hunk 1]
```diff
@@ -11,16 +11,19 @@ import AddIcon from "@material-ui/icons/PlaylistAdd";
...
   constructor(props) {
     super(props);
     this.state = {
       data: this.props.post,
       isFav: false,
-      imageSrc: `/images/` + this.props.post.name + `.png`,
+      imageSrc: this.props.post.imageSrc,
       showDetails: false,
-      isLogginIn: false
+      isLogginIn: false,
```
Comment (nicolehli) [inline imports/ui/components/Card.jsx]: What is this for? Only place declared is here.

Comment (nicolehli) [inline imports/ui/components/Card.jsx]: What is this for? Only place declared is here.

File: imports/ui/components/FreshForm.jsx [hunk 2]
```diff
@@ -44,7 +45,32 @@ class FreshForm extends Component {
...
+        this.setState({
+          imageSrc: "/images/" + this.state.name + ".png"
+        });
+      };
+    } else {
+      // Image was selected then unselected, need to revert to base image.
+      this.setState({
+        imageSrc: "/images/" + this.state.name + ".png"
+      });
+    }
```
Comment (nicolehli) [inline imports/ui/components/FreshForm.jsx]: Just the setting image part. Let you rewrite to have uploaded image take priority (e.g. uploaded picture of image then set Blueberries name, should show uploaded image not default)

File: imports/ui/components/Card.jsx (hunk 1 above)
Comment (YaoLiu93) [reply imports/ui/components/Card.jsx]: Not sure... removed!

File: imports/ui/components/FreshForm.jsx (hunk 2 above)
Comment (YaoLiu93) [reply imports/ui/components/FreshForm.jsx]: Changed prioirty to always use uploaded image.

**Pull request merged**